### ISSUE TITLE
About mAP

### ISSUE THREAD

**Issue by shinemrl** [TreB1eN/Pytorch0.4.1_Openpose#5]
Thanks a lot for your greak work!
Did you calculate the mAP of the model? I haven't seen any code about that so I wonder how you measure the performance of your model.
Could you give me some information?
Thank you

**Comment by anerisheth19** [comment]
> Thanks a lot for your greak work!
> Did you calculate the mAP of the model? I haven't seen any code about that so I wonder how you measure the performance of your model.
> Could you give me some information?
> Thank you

I have the same concern. If the training accuracy is not reported, how can one use the pretrained model?
